packages/editor/package.json

```diff
@@ -128,6 +125,7 @@
     "nodemon": "^2.0.1",
     "npm-run-all": "^4.1.5",
     "shx": "^0.3.2",
+    "snapshot-diff": "^0.9.0",
```

Comment:
Allows to only snapshot the difference. Reduces the size of the snapshots quite a bit